src/components/ImageList/ImageList.component.tsx

```
@@ -0,0 +1,70 @@
+    return (
+        <Box>
+            <StyledImageList
+                variant='standard'
+                cols={isTabletOrSmaller ? 1 : 3}
+                gap={isTabletOrSmaller ? 16 : 45}
+                rowHeight={300}
+            >
+                {[...IMAGE_LIST_CONFIG]
+                    .sort(
+                        (a, b) =>
+                            (a.displayOrder ?? Infinity) -
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)